Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -362,7 +362,29 @@ public class RepositoryPackageContainer: BasePackageContainer, CustomStringConve
         // Compute the map of known versions.
         //
         // FIXME: Move this utility to a more stable location.
-        let knownVersions = Git.convertTagsToVersionMap(repository.tags)
+        let knownVersionsWithDuplicates = Git.convertTagsToVersionMap(repository.tags)
+        
+        let knownVersions = knownVersionsWithDuplicates.mapValues { (tags) -> String in
```

Comment:
This can be simplified to:

```swift
let knownVersions = knownVersionsWithDuplicates.mapValues({ tags in
    // Return the non v-prefixed version in case of duplicates.
    if tags.count == 2 {
        // FIXME: We need to warn if the two tags point to different git ref.
        return tags.first(where: { $0.hasPrefix("v") })!
    }
    assert(tags.count == 1, "Unexpected number of tags")
    return tags[0]
})
```